tests/PHPStan/Analyser/data/integer-range-types.php

```diff
@@ -218,70 +218,70 @@ public function math($i, $j, $z, $pi, $r1, $r2, $r3, $rMin, $rMax, $x, $y) {
 		assertType('int<2, 13>', $r1 + $j);
 		assertType('int<-2, 9>', $r1 - $j);
 		assertType('int<1, 30>', $r1 * $j);
-		assertType('float|int<0, 10>', $r1 / $j);
+		assertType('float|int<1, 10>', $r1 / $j);
```

Comment:
0 was never possible

Reply:
Please verify each expectation change on wolfram alpha like
https://www.wolframalpha.com/input?i=%28interval%5B1%2C10%5D+%2F+interval%5B1%2C3%5D%29

Reply:
@staabm I checked every expectation, and I'm good with my results.
 I added some link in the review, but wolfram alpha is sometimes wrong.

See https://www.wolframalpha.com/input?i=%28-1%2Finterval%5B-9999%2C+5%5D+%29
or https://www.wolframalpha.com/input?i=%28interval%5B1%2C10%5D+%2F+interval%5B-9999%2C5%5D%29

This is especially why in the code I split the `/` computation to not have together the positive and the negative int.